src/csharp/Grpc.IntegrationTesting/Control.cs

```diff
@@ -152,6 +153,7 @@ public enum ServerType {
     /// used for some language-specific variants
     /// </summary>
     [pbr::OriginalName("OTHER_SERVER")] OtherServer = 3,
+    [pbr::OriginalName("CALLBACK_SERVER")] CallbackServer = 4,
```

Comment:
Not sure why these changes and line endings changes were generated when I ran `generate_proto_csharp.sh`, @jtattermusch ?

Reply:
There might have been some additions changes to the `.proto` file and `.cs` hasn't been regenerated since then. (generally this is fine).